.gitignore

```diff
@@ -88,6 +88,11 @@ docGen/
 
 # Release outputs
 release/
+
+# e2e tests outputs
 e2e-tests/test-results/
 e2e-tests/playwright-report/
 e2e-tests/playwright/.cache/
+
+# Snapshots generated while running e2e tests locally
```

Comment:
Let's also expound on why we don't want to commit them. Could be something along these lines:

`Screenshots differ between browsers and platforms due to different rendering, fonts and more. Your Windows development environment may differ from someone else's. For more information, visit https://playwright.dev/docs/test-snapshots`